### PR TITLE
qt5 clangarm64, part 2

### PR DESCRIPTION
**Pull request by driver1998**
Enable everything except:
- qt5-doc
- qt5-static
- ~~qt5-connectivity, it link fails because `bthprops.lib` is missing in MinGW/ARM64.~~

### CODE REVIEW

**Comment by Biswa96** [comment]
> it link fails because bthprops.lib is missing in MinGW/ARM64.

It can be added in mingw-w64 in a second 😉 But a little confusion. Is the shared library is a control panel module, named bthprops.cpl? MS docs says that it's a DLL, named bthprops.dll.

**Comment by jeremyd2019** [comment]
my system only has a bthprops.cpl

**Comment by driver1998** [comment]
> It can be added in mingw-w64 in a second 😉 But a little confusion. Is the shared library a control panel module, named bthprops.cpl? MS docs says that it's a DLL, named bthprops.dll.

It seems to be:
https://stackoverflow.com/questions/19436462/loadlibrarybthprops-dll-fails-file-not-found

**Comment by Biswa96** [comment]
I have sent a patch in mingw-w64 mailing list to add bthprops in arm and arm64 here https://sourceforge.net/p/mingw-w64/mailman/message/37630872/

**Comment by Biswa96** [comment]
bthprops for arm was added in this upstream commit https://sourceforge.net/p/mingw-w64/mingw-w64/ci/bb521678353b31f0c88fa6a931eed5595f051510/

**Comment by jeremyd2019** [comment]
Updated mingw-w64 packages with bthprops for arm should be in the repos now.

**Comment by driver1998** [comment]
Let's see if it will work!

**Comment by jeremyd2019** [comment]
ARM64 CI run failed:
https://github.com/msys2-arm/MINGW-packages/runs/5801093566?check_suite_focus=true
```
  [MSYS2 CI] mingw-w64-qt5-webglplugin: Building binary
  
  ==>  MINGW_ARCH: clangarm64
    -> Building clangarm64...
  ==> Making package: mingw-w64-qt5-webglplugin 5.15.3-1 (Sat, Apr  2, 2022  9:03:23 PM)
  ==> Checking runtime dependencies...
  ==> Checking buildtime dependencies...
  ==> Installing missing dependencies...
  error: target not found: mingw-w64-clang-aarch64-qt5-websockets
```

File: mingw-w64-qt5-websockets/PKGBUILD [hunk 1]
```diff
@@ -9,7 +9,7 @@ pkgver=${_qtver}+kde+r3
 pkgrel=1
 _commit=7196d2cc34adf9f45b50a9488f4ff95b36092993
 arch=('any')
-mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm32')
```
Comment (jeremyd2019) [inline mingw-w64-qt5-websockets/PKGBUILD:12]:
```suggestion
mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
```

**Comment by jeremyd2019** [comment]
Successful ARM64 CI run: https://github.com/msys2-arm/MINGW-packages/runs/5890673759?check_suite_focus=true

**Pull request merged**